### ISSUE TITLE
Windows 11 style PaneDisplayMode

### ISSUE THREAD

**Issue by tobiasht** [bdlukaa/fluent_ui#82]
![image](https://user-images.githubusercontent.com/20655261/137390779-8515a273-a0af-4771-a710-c0be7fa5e257.png)

I really like the navigation UI seen in the new Windows 11 Store application. This is very similar to the compact display mode, with three minor adjustments, more spacing, space for labels, and does not expand.

Would you be interested in adding `PaneDisplayMode.win11` as an additional display mode?

**Comment by bdlukaa** [comment]
I don't think adding a new display mode is a valid request. Natively, on XAML, we only have the ones we have in this package.

I think a better request would maybe be "Custom pane width"? The padding can be custom using
https://github.com/bdlukaa/fluent_ui/blob/master/lib/src/controls/navigation/navigation_view/style.dart#L83

**Comment by tobiasht** [comment]
Yeah, that sounds like a good solution. Didn´t know about the `iconPadding` parameter. Is there a way we could easily add a label to the `PaneDisplayMode.Compact`?
